base/scripts/code.game.cs

```diff
@@ -286,7 +286,6 @@ function UpdateClientTimes(%time)
   
   Game::UpdateClientScores();
   
-  zadmin::ActiveMessage::All(MatchStarted);

```

Comment:
i added this - could you put this back? you might be able to just pull in upstream, im not sure